docs/api/vast-client.md

```diff
@@ -304,3 +360,72 @@ const vastParser = vastClient.getParser();
 // Clear the url template filters used
 vastParser.clearUrlTemplateFilters();
 ```
+
+### addURLTemplateFilter(filter)
+
+Adds a filter function to the array of filters which are called before fetching a VAST document.
+
+#### Parameters
+
+- **`filter: function`** - The filter function to be added at the end of the array
+
+#### Example
+
+```Javascript
+vastClient.addURLTemplateFilter( vastUrl => {
+    return url.replace('[DOMAIN]', 'mywebsite.com')
+});
+
+/*
+For a VASTAdTagURI defined as :
+<VASTAdTagURI>http://example.dailymotion.com/vast.php?domain=[DOMAIN]</VASTAdTagURI>
+HTTP request will be:
+http://example.dailymotion.com/vast.php?domain=mywebsite.com
+*/
+```
+
+### removeURLTemplateFilter()
```

Comment:
As it has been replaced with removeLastURLTemplateFilter you need to update it aswell